pages/clusters/queue_metrics.md

```diff
@@ -0,0 +1,24 @@
+# Queue metrics
```

Comment:
I think we should include "clusters" in the page title to help with on-site search results.

```suggestion
# Queue metrics in clusters
```

(This is just an update to the page title, not the nav)